python-backend/main.py

```diff
@@ -44,19 +42,6 @@
                    allow_headers=["*"])
 
 
-def parse(data: Input) -> tuple[sympy.core, sympy.core, sympy.core, Symbol]:
```

Comment:
This function was moved to the input.py file above.

Reply:
These comments are helpful!